### PR TITLE
Started on some simple console output coloring

### PR DESCRIPTION
**Pull request by johnmcdowall**
WIP :bomb: 

DO NOT MERGE :bomb:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,28 +4,29 @@
   "version": "0.0.14",
   "description": "truffle - Simple development framework for Ethereum",
   "dependencies": {
-    "coffee-script": "*",
```
Comment (tcoulter) [inline package.json]: John, I'm having trouble noticing what you did here. Did you remove any previous dependencies, or just reorder them? (I see you added some; that's fine)

Comment (johnmcdowall) [inline package.json]: All I did was npm install colors --save.  Thanks npm. 

PS. Don't merge this quite yet, updated the description to indicate its a WIP.

**Comment by tcoulter** [comment]
Alright. I just merged with master as I pushed a new change that caused a conflict.

**Comment by tcoulter** [comment]
@johnmcdowall You still interested in this PR? If not, I'd give it a close. Cheers!

**Comment by tcoulter** [comment]
Closing for housekeeping, as it's a little outdate. However, if there's significant interest let's reopen!

**Pull request closed without merging**